nservicebus/sqlserver/multiple-databases.md

```diff
@@ -65,3 +72,16 @@ and the following connection strings:
 <!-- import sqlserver-multidb-connectionstrings -->
 
 the messages sent to `billing` will go to database `Billing` on server `DbServerB` while the messages to `sales` will go to the database and server set by default i.e. `MyDefaultDB` on server `DbServerA`.
+
+
+### ServiceControl
+
+In order to use a single instance of ServiceControl for the whole system when there are multiple databases used for queues, the `audit` and `error` queues need to be present in a single dedicated database (the ServiceControl database). In addition to configuring the mapping between the user endpoints and connection strings (as described above), there must be a mapping that redirects `audit` and `error` pseudo-endpoints to the ServiceControl database.
```

Comment:
how about this

> ServiceControl requires the error and audit queues that it ingests from to be presented on a single dedicated database. In a system where there are multiple databases, used by various endpoints, this can be achieved by redirecting the queues using the following configuration endpoint configuration: